Resources/ServerInfo/Guidebook/SpaceStation14.xml

```diff
@@ -1,14 +1,15 @@
-﻿<Document>
+<Document>
 # Space Station 14
 
-Welcome to the pre-alpha of Space Station 14! We hope you enjoy the game, and this entry will serve to guide you on how to learn to play. There's quite a bit to read, but SS14 is in itself a very large and in-depth game, and hopefully these guides help you enjoy it to it's fullest.
+This entry marks the beginning of the advanced guidebook entries. Most information found here only applies to offical Wizard's Den servers, or servers that are similar to it.
```

Comment:
It would be better to word this in a way that makes it clear that other servers can (and should) have their own guidebook modifications.

Reply:
good idea. will do